Lib/test/inspect_fodder.py

```diff
@@ -91,3 +91,24 @@ def as_method_of(self, obj):
 
 custom_method = Callable().as_method_of(42)
 del Callable
+
+# line 95
+class WhichComments:
+  # line 97
+    # before f
+    def f(self):
+      # line 100
+        # start f
+        return 1
+        # line 103
+        # end f
+       # line 105
+    # after f
+
+    # before asyncf - line 108
+    async def asyncf(self):
+        # start asyncf
+        return 2
+        # end asyncf
+       # after asyncf - line 113
+  # after WhichComments - line 114
```

Comment:
Perhaps add a trailing comment for the class block which should be included?
```suggestion
    # end of WhichComments - line 114
  # after WhichComments - line 115
```

Reply:
Good catch. I'll commit that (needs an update in the expected end line # in test_inspect as well).